sklearn/calibration.py

```diff
@@ -328,6 +348,8 @@ def fit(self, X, y, sample_weight=None):
                     this_estimator.fit(X, y, sample_weight)
```

Comment:
It seems we should avoid positional arguments here. If not, we get an error with SGDClassifier for example (third positional argument is coef_init and not sample_weight).
`this_estimator.fit(X = X, y = y, sample_weight = sample_weight)`

Reply:
Yes, good catch, I fixed this line.